HW_3.py

```diff
@@ -0,0 +1,38 @@
+import re
+
+text = """homEwork:
+
+  tHis iz your homeWork, copy these Text to variable.
+
+
+
+  You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
```

Comment:
...add it to the END OF this Paragraph.

The small inconsistency here: you added a sentence with last words to the end of text.